src/__tests__/patients/related-persons/AddRelatedPersonModal.test.tsx

```diff
@@ -42,106 +39,94 @@ describe('Add Related Person Modal', () => {
-      expect(relationshipTypeTextInput.prop('label')).toEqual(
-        'patient.relatedPersons.relationshipType',
+      setup()
+
+      const relationshipTypeInput = screen.getByLabelText(
+        /^patient.relatedPersons.relationshipType$/i,
       )
+      expect(relationshipTypeInput).toBeInTheDocument()
+      expect(relationshipTypeInput).not.toBeDisabled()
     })
 
     it('should render a cancel button', () => {
-      const wrapper = setup()
-      const cancelButton = wrapper.findWhere(
-        (w: { text: () => string }) => w.text() === 'actions.cancel',
-      )
+      setup()
 
-      expect(cancelButton).toHaveLength(1)
+      expect(screen.getByRole('button', { name: /close/i })).toBeInTheDocument()
     })
 
     it('should render an add new related person button button', () => {
-      const wrapper = setup()
-      const modal = wrapper.find(Modal) as any
-      expect(modal.prop('successButton').children).toEqual('patient.relatedPersons.add')
+      setup()
+
+      expect(
+        screen.getByRole('button', { name: /patient.relatedPersons.add/i }),
+      ).toBeInTheDocument()
     })
 
     it('should render the error when there is an error saving', async () => {
-      const wrapper = setup()
+      setup()
+
+      const expectedErrorMessage = 'patient.relatedPersons.error.unableToAddRelatedPerson'
       const expectedError = {
-        message: 'patient.relatedPersons.error.unableToAddRelatedPerson',
         relatedPersonError: 'patient.relatedPersons.error.relatedPersonRequired',
         relationshipTypeError: 'patient.relatedPersons.error.relationshipTypeRequired',
       }
-
-      await act(async () => {
-        const modal = wrapper.find(Modal)
-        const onSave = (modal.prop('successButton') as any).onClick
-        await onSave({} as React.MouseEvent<HTMLButtonElement>)
-      })
-      wrapper.update()
-
-      const alert = wrapper.find(Alert)
-      const typeahead = wrapper.find(Typeahead)
-      const relationshipTypeInput = wrapper.find(TextInputWithLabelFormGroup)
-
-      expect(alert.prop('message')).toEqual(expectedError.message)
-      expect(alert.prop('title')).toEqual('states.error')
-      expect(typeahead.prop('isInvalid')).toBeTruthy()
-      expect(relationshipTypeInput.prop('isInvalid')).toBeTruthy()
-      expect(relationshipTypeInput.prop('feedback')).toEqual(expectedError.relationshipTypeError)
+      expectOneConsoleError(expectedError)
+
+      userEvent.click(screen.getByRole('button', { name: /patient.relatedPersons.add/i }))
+      expect(await screen.findByRole('alert')).toBeInTheDocument()
+      expect(screen.getByText(expectedErrorMessage)).toBeInTheDocument()
+      expect(screen.getByText(/states.error/i)).toBeInTheDocument()
+      expect(screen.getByPlaceholderText(/^patient.relatedPerson$/i)).toHaveClass('is-invalid')
+      expect(screen.getByLabelText(/^patient.relatedPersons.relationshipType$/i)).toHaveClass(
```

Comment:
I made a change in Dashboard.tsx to test something else. I eventually reverted to code back to what it is currently. Rerunning tests failed 4 tests including this one.

![image](https://user-images.githubusercontent.com/5468546/103901864-63bc4780-5102-11eb-86cb-17c41c72ea88.png)

![image](https://user-images.githubusercontent.com/5468546/103902431-32904700-5103-11eb-9af5-19fca1c33257.png)

Reply:
a rerun of all tests passed so not sure. Perhaps it was just a temporary failure.

Reply:
Some of the tests are reliant on some fickle environment situations, not sure what the change was but we can discuss that further maybe it will help us narrow down the cause.

Reply:
> Some of the tests are reliant on some fickle environment situations, not sure what the change was but we can discuss that further maybe it will help us narrow down the cause. @nobrayner

The change was in src/dashboard/Dashboard.tsx line 12, I changed the h3 text to something to verify if the dashboard tests would fail. 

You mention the environment, perhaps you are onto something as I did this quick test via gitpod so maybe could explain why I got the failures for the unrelated tests?

Reply:
These tests that are failing due to a timeout may have been toeing the line already, and then using a slower environment could cause them to take too long.